src/test/indexTest.ts

```diff
@@ -58,6 +59,13 @@ describe('index', () => {
             });
         }).timeout(5000);
 
+        it('validate importValue flag', (done) => {
+            exec('node lib/index.js validate testData/valid/yaml/valid_import_value_type.yaml --importValues ImportedValueOutputPort="20"', function(error, stdout, stderr) {
```

Comment:
What if the value is not quoted? Does this handle plain integers well?

Reply:
I aped the importValues handling code from the code for parameters, so it should work the same.  I added a test that confirms.